inst/templates/bs4_book.html

```diff
@@ -122,6 +122,25 @@ <h2>On this page</h2>
     document.getElementsByTagName("head")[0].appendChild(script);
   })();
 </script>
+<!-- https://stackoverflow.com/a/49687594 -->
```

Comment:
How do you generally acknowledge using code from SO?

Reply:
Seems like licence is [CC-BY-SA-4.0](https://creativecommons.org/licenses/by-sa/4.0/) - https://stackoverflow.com/help/licensing

So it should be applied especially if you copy-pasted without change (and not just inspiration). 

@yihui do you have a usual practice for this ?

Reply:
I rarely copy and paste code directly from SO (usually I try to understand the code and rewrite it by myself). If you copy, the requirements are listed in the license: https://creativecommons.org/licenses/by-sa/3.0/ For attribution, a link might be enough; for ShareAlike, bookdown's license (GPL-3) should be compatible.

Reply:
@maelle looking at the code and SO question: it seems it is already rewritten right ? 
Not juts copy pasted but adapted for `bs4_book()`

Reply:
Yes @mine-cetinkaya-rundel modified it.